src/helps/cron.job.ts

```diff
@@ -0,0 +1,27 @@
+import cron from "node-cron";
+import { db } from "../database/models/index";
+import sendPasswordUpdateNotification from "../utils/sendChangePasswordNofication";
+import dotenv from "dotenv";
+
+dotenv.config();
+
+const startCronJob = (): void => {
+    const cronSchedule = process.env.CRON_SCHEDULE || '* * * * *';
+    const passwordExpirationPeriodMinutes = parseInt(process.env.PASSWORD_EXPIRATION_PERIOD_MINUTES || '1', 10);
+
+    cron.schedule(cronSchedule, async () => {
+        const users = await db.User.findAll();
+        
+        users.forEach(async (user: any) => {
+            const lastPasswordChangeDate: Date = user.passwordLastChanged;
```

Comment:
I am not seeing where this field is being changed. How is it helping you to know when is the password last updated?

Reply:
It has been added to the user model. It will also be added to the token on login. It will also be updated on updating the password.